.github/actionlint.yaml

```diff
@@ -1,4 +1,4 @@
-self-hosted-runner:
+runnerssted-runner:
```

Comment:
Why this change?

Reply:
This looks like a typo that crept in, sorry.